src/enums/cacheTimes.ts

```diff
@@ -0,0 +1,5 @@
+export enum CacheTimes {
+  SHORT = 5, // Keep data for 5 seconds
```

Comment:
use constants rather than enum, just need only 1 time, put the unit in the name
exp: CACHE_TIME_MS = 10

Reply:
done